frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -344,4 +348,4 @@ class Dashboard extends Component {
   }
 }
 
-export default DashboardControls(Dashboard);
+export default _.compose(DashboardControls, withAutoApplyFilters)(Dashboard);
```

Comment:
A lot of logic are implemented using hooks, so I need to use a functional component, but `<Dashboard />` is a class component, so it's easier at this point to use HoC than to refactor the `<Dashboard />` to a functional component.